ursine-umbra.scss

```diff
@@ -1119,6 +1119,9 @@ footer.ty-footer {
     -moz-background-size: cover;
     -o-background-size: cover;
     opacity: 0;
+    -webkit-transition: opacity 0.2s ease-out;
+    -o-transition: opacity 0.2s ease-out;
```

Comment:
Probably don't need the vendor prefixes

Reply:
Removed, but only for my previous commit. Should you consider open another issue or pr for removing other vendor prefixed rules for the original branch?